### PR TITLE
Add input to disable metrics

### PR DESCRIPTION
**Pull request by bryanhonof**
By default, Flox sends out metrics, regardless if it's running in CI or not. This change allows the user to specify that they don't want Flox to send out periodic metrics.

Closes flox/install-flox-action#93

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -81,11 +81,17 @@ jobs:
...
+          disable-metrics: true
+
 
       - name: "Test: flox --version"
         run: |
           flox --version
 
+      - name: "Test: flox config"
+        run: |
+          flox config
```
Comment (garbas) [inline .github/workflows/ci.yml:93]: Can we make this a test that checks if disable-metrics is set correctly in config?

File: .github/workflows/update-dist.yml [hunk 2]
```diff
@@ -28,7 +28,7 @@ jobs:
     runs-on: ubuntu-latest
 
     permissions:
-      contents: read
```
Comment (garbas) [inline .github/workflows/update-dist.yml:31]: why is this change needed?

File: package.json [hunk 3]
```diff
@@ -31,7 +31,7 @@
     "ci-test": "jest",
     "format:write": "prettier --write **/*.js",
     "format:check": "prettier --check **/*.js",
-    "package": "ncc build src/index.js --license licenses.txt",
+    "package": "ncc build src/index.js --license LICENSE",
```
Comment (garbas) [inline package.json:34]: Why is this change needed? I don't think we have licenses.txt file.

File: .github/workflows/update-dist.yml (hunk 2 above)
Comment (bryanhonof) [reply .github/workflows/update-dist.yml:31]: Otherwise, the action that generates the dist folder fails to push to the repo.
https://github.com/flox/install-flox-action/pull/98/commits/24ff7b6d68759227cff22a53ddd75663c9c3a6ed

File: package.json (hunk 3 above)
Comment (bryanhonof) [reply package.json:34]: The license file we have in the GitHub repo is called `LICENSE`, we indeed do not have a `licenses.txt`. Unless I'm misinterpreting the flag, this just adds that `LICENSE` file to the dist folder.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (bryanhonof) [reply .github/workflows/ci.yml:93]: Sure.

File: package.json (hunk 3 above)
Comment (garbas) [reply package.json:34]: yes, lets revert this change

```suggestion
    "package": "ncc build src/index.js --license LICENSE",
```

**Review by garbas — APPROVED** [review]
